CHANGELOG_v4_draft.md

```diff
@@ -0,0 +1,117 @@
+(note: this will be moved to packages/server/CHANGELOG.md as part of the actual release process)
+
+## v4.0.0
+
+### BREAKING CHANGES
+
+Apollo Server contains quite a few breaking changes: most notably, a brand new package name! Read our [migration guide](https://www.apollographql.com/docs/apollo-server/migration/) for more details on how to update your app.
+
+
+#### Bumped dependencies
+
+The minimum versions of these dependencies have been bumped to provide an improved foundation for the development of future features.
+
+- Dropped support for Node.js v12, which is no longer under [long-term support](https://nodejs.org/en/about/releases/#releases) from the Node.js Foundation.
+- Dropped support for versions of the `graphql` library prior to `v16.6.0`.
+  - Upgrading `graphql` may require you to upgrade other libraries that are installed in your project. For example, if you use Apollo Server with Apollo Gateway, you should upgrade Apollo Gateway to at least v0.50.1 or any v2.x version for full `graphql` 16 support before upgrading to Apollo Server 4.
+- If you use Apollo Server with TypeScript, you must use TypeScript v4.7.0 or newer.
+
+
+#### New package structure
+
+Apollo Server 4 is distributed in the `@apollo/server` package. This package replaces `apollo-server`, `apollo-server-core`, `apollo-server-express`, `apollo-server-errors`, `apollo-server-types`, and `apollo-server-plugin-base`.
+
+The `@apollo/server` package exports the `ApolloServer` class. In Apollo Server 3, individual web framework integrations had their own subclasses of `ApolloServer`. In Apollo Server 4, there is a single `ApolloServer` class; web framework integrations define their own functions which use a new stable integration API on `ApolloServer` to execute operations.
+
+Other functionality is exported from "deep imports" on `@apollo/server`. `startStandaloneServer` (the replacement for the batteries-included `apollo-server` package) is exported from `@apollo/server/standalone`. `expressMiddleware` (the replacement for `apollo-server-express`) is exported from `@apollo/server/express4`. Plugins such as `ApolloServerPluginUsageReporting` are exported from paths such as `@apollo/server/plugin/usageReporting`.
```

Comment:
We could add links to the API docs for `startStandaloneServer` and `expressMiddleware` here to give folks more information on those two functions, but we could also not 🤷‍♀️

Reply:
I kind of feel like folks need to hit up the migration docs anyway rather than the reference docs...